src/js/feedback/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AlertBox from '../../common/components/AlertBox';
+import ErrorableTextarea from '../../common/components/form-elements/ErrorableTextarea';
+import ErrorableTextInput from '../../common/components/form-elements/ErrorableTextInput';
+import ErrorableCheckbox from '../../common/components/form-elements/ErrorableCheckbox';
+
+class FeedbackForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      suppressDescriptionErrors: true,
+      suppressEmailErrors: true
+    };
+  }
+
+  setDescription = ({ value: description, dirty }) => {
+    if (dirty) this.setState({ suppressDescriptionErrors: false });
+    this.props.setFormValues({ description });
+  }
+
+  setEmail = ({ value: email, dirty }) => {
+    if (dirty) this.setState({ suppressEmailErrors: false });
+    this.props.setFormValues({ email });
+  }
+
+  sendFeedback = (event) => {
+    event.preventDefault();
+    this.props.sendFeedback(this.props.formValues);
+  }
+
+  descriptionErrorMessage = () => {
+    return !this.state.suppressDescriptionErrors ? this.props.formErrors.description : '';
+  }
+
+  emailErrorMessage = () => {
+    return !this.state.suppressEmailErrors ? this.props.formErrors.email : '';
+  }
+
+  render() {
+    return (
+      <form className="feedback-form" onSubmit={this.sendFeedback}>
+        <h4 className="feedback-widget-title">Tell us what you think</h4>
+        <div className="va-flex">
+          <div className="feedback-widget-description-container">
+            <ErrorableTextarea
+              label="What can we do to make Vets.gov better?"
```

Comment:
It looks like the class for this label's asterisk (`hca-required-span`) is inconsistent with that of the email label's asterisk (`form-required-span`). This inconsistency is not introduced by these changes, but it might make sense to address it along with them.

Reply:
Also, should the tab order be to go through the form and then to go to the contact information?

Reply:
I would rather not dig into the errorable inputs. I'm not the author and this PR is heating up to be pushed through soon. Maybe @jbalboni has some input?

The tab order needs to be fixed though. I don't think the contact info is a big deal because it actually switches order on mobile, but the form itself should probably show/hide instead of an actual render.